src/gtk/textentry.cpp

```diff
@@ -272,6 +272,12 @@ DoHandleClipboardCallback( GtkWidget *widget,
         // ourselves in the event handler
         g_signal_stop_emission_by_name (widget, signal_name);
     }
+    wxTextCtrl *text = dynamic_cast<wxTextCtrl *>( win );
```

Comment:
There should be no need for dynamic casts, these classes provide all the virtual functions you need to avoid them.